tests/test-run.sh

```diff
@@ -149,6 +149,17 @@ assert_file_has_content json-status.json '"child-pid": [0-9]'
 assert_file_has_content_literal json-status.json '"exit-code": 42'
 echo "ok info and json-status fd"
 
+DATA=$($RUN --proc /proc --unshare-all --info-fd 42 --json-status-fd 43 -- bash -c 'stat -L --format "%n %i" /proc/self/ns/*' 42>info.json 43>json-status.json 2>err.txt)
+
+for NS in "cgroup" "ipc" "mnt" "net" "pid" "uts"; do
```

Comment:
We need the cgroup check to be optional here (as it might not be supported in older kernels). Or just drop it.